index.html

```diff
@@ -125,15 +125,18 @@ <h2 class="fw-bold">Take a look inside</h2>
             <div class="row justify-content-between">
               <div class="col-md-6 text-white align-self-center text-center text-md-left my-2">Copyright © 2021 Files
                 Community</div>
-              <div class="col-md-6 align-self-center text-center text-md-right my-2" id="social-media">
-                <a href="https://github.com/files-community" class="d-inline-block text-center ml-2" aria-label="View the Files community GitHub organization">
-                  <i class="bi-github"></i>
-                </a>
-                <a href="https://discord.gg/mr5hVu8" class="d-inline-block text-center ml-2" aria-label="Visit the Files community Discord server">
-                  <i class="bi-discord"></i>
-                </a>
+                <div class="col-md-6 align-self-center text-center text-md-right my-2" id="social-media">
+                  <div class="icons-container">
+                    <a href="https://github.com/files-community" rel="external" class="d-inline-block ml-2">
+                      <i class="bi-github"></i>
+                    </a>
+                    <a href="https://discord.gg/mr5hVu8" rel="external" class="d-inline-block ml-2">
+                      <i class="bi-discord"></i>
+                    </a>
+                    <a href="http://paypal.me/yaichenbaum" rel="external" class="d-inline-block text-center ml-2 icon-paypal"></a>
```

Comment:
Adding rel="external" allows search engines and assistive software to know how to treat this link correctly (external meaning that this is a different website, not something on our website).